### DISCUSSION TITLE
Browse possibly isn't working?

### DISCUSSION THREAD

**Discussion by illettante** [MusicPlayerDaemon/MPD#1474] [Q&A] [answered]
Should I see a list of directories in the Browse screen? Currently it displays nothing.

**Comment by MaxKellermann** [comment]
Maybe. Do directories exist in your database? You gave no information, except for "browse screen displays nothing", but what browse screen and what does this have to do with MPD?
https://www.musicpd.org/help/

**Comment by illettante** [comment]
Yep, you're right, my apologies, more info needed. Sorry, i clicked a link for ncmpc support and blithely assumed i'd ended up at a specifically ncmpc-related repo. That'll teach me to open my eyes before I open my mouth!
The 'browse screen' in question is the '3:Browse' screen in ncmpc. There should be no files, one directory in there, currently there's nothing. Locally hosted instance of MPD, running as user.

**Comment by illettante** [comment, marked as answer]
Found the answer, my bad - switched from the old database to the new database in my .conf and didn't update the location to reflect the location of my actual database. Facepalm. Thanks for patience.
